tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyMain.java

```diff
@@ -2823,6 +2825,24 @@ private void doStartup() {
                                     + phaseString);
                 }
                 currentPhase = newPhase;
+                String ddlPhaseString = survprops.getProperty("CLDR_DDL_PHASE", null);
+                Phase ddlPhase = null;
+                try {
+                    if (ddlPhaseString != null && !ddlPhaseString.isEmpty()) {
+                        if (currentPhase == Phase.READONLY) {
+                            busted(
+                                    "Error: Cannot have a CLDR_DDL_PHASE when CLDR_PHASE=READONLY. Remove the CLDR_DDL_PHASE.");
```

Comment:
maybe allow CLDR_DDL_PHASE=READONLY when CLDR_PHASE=READONLY?

Reply:
Could, I was trying to simply have that unset when the ST is in readonly. Otherwise DDL would remain READONLY when the rest of it  reopens.

Reply:
good point